src/components/header/header.tsx

```diff
@@ -1,23 +1,25 @@
 import { useAtomValue, useSetAtom } from "jotai";
-import { authAtom, userProfileAtom } from "../../../atoms/auth";
+import { authAtom, userProfileAtom } from "../../atoms/auth";
 import { useEffect, useState } from "react";
-import { useApi } from "../../../hooks/use-api";
-import { PersonTotalTime } from "../../../generated/client";
-import { errorAtom } from "../../../atoms/error";
-import LoaderWrapper from "../../generics/loader-wrapper";
-import { getHoursAndMinutes } from "../../../utils/time-utils";
+import { useApi } from "../../hooks/use-api";
+import { PersonTotalTime } from "../../generated/client";
+import { errorAtom } from "../../atoms/error";
+import LoaderWrapper from "../generics/loader-wrapper";
+import {  Container, Grid } from "@mui/material";
+import BalanceCard from "./balance-card";
+import HomeNav from "./nav";
 
 /**
  * Dashboard screen component
  */
-function DashboardScreen () {
+function Header () {
```

Comment:
Header is a fine name for now but we should think what this component is actually going to be doing in the future, is it going to become a tile for balance/ timebank for example. Perhaps no need to change now but something to keep in mind.